import axios from "axios"

const exceptionHandler = (error: any, message: string) => {
  if (axios.isAxiosError(error)) {
    return `ðŸ˜« error message \n ${message} \n ${error.message}`
  }
  return `ðŸ˜± unexpected error: ${error}`
}

export { exceptionHandler }
